social_media/accounts/serializer.py

```diff
@@ -0,0 +1,46 @@
+from rest_framework import serializers
+
+from .models import User
+from datetime import datetime
+
+from upload.models import Posts
+from upload.serializer import PostSerializer
+
+
+class AccountRegistrationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ['username', 'email', 'bio', 'password']
+
+    def create(self, validated_data):
+        user = User(
```

Comment:
Don't do this. Instead make sure validated_data has all the keys and values and then pass it like kwargs:

`User.objects.create(**validated_data)`